tests/test_export_caffe2.py

```diff
@@ -18,6 +20,8 @@
 @unittest.skipIf(os.environ.get("CIRCLECI"), "Caffe2 tests crash on CircleCI.")
 class TestCaffe2Export(unittest.TestCase):
```

Comment:
can you add another `@unittest.skipIf` instead of putting it inside setUp? Seems nice to be consistent.